featuretools/primitives/base/primitive_base.py

```diff
@@ -34,6 +34,14 @@ class PrimitiveBase(object):
     # (bool) If True will only make one feature per unique set of base features
     commutative = False
 
+    def __call__(self, data):
```

Comment:
this won't handle primitives that requires to input variables. We should use `*args` and `**kwargs`